### PR TITLE
[ADD] Debug logs added

### PR DESCRIPTION
**Pull request by techpool**
Debug logs for the following events has been added:
1. connection
2. name
3. join
4. msg
5. disconnect

### CODE REVIEW

**Comment by techpool** [comment]
@mbrandau does this make sense?

File: web_server.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ io.on('connection', function(socket) {
     //New message
     socket.on('msg', msg => {
 
-      log.debug(`EVENT: msg, MSG: Triggered by - ${socket.meta.username} MSG - ${msg}`);
+      log.debug(`${name} (${socket.request.connection.remoteAddress}) has sent a new message`);
```
Comment (mbrandau) [inline web_server.js]: Please log the message too

File: web_server.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ io.on('connection', function(socket) {
     // Handle disconnect
     socket.on('disconnect', reason => {
 
-      log.debug(`EVENT: disconnect, MSG: Triggered by - ${socket.meta.username} MSG - User has been disconnected from the live connection`);
+      log.debug(`${name} (${socket.request.connection.remoteAddress}) has been disconnected`);
```
Comment (mbrandau) [inline web_server.js]: Can you add the reason, please?

**Comment by mbrandau** [comment]
@techpool What do you mean by that? Logging the messages in a better readable format or your changes? :smile:

**Comment by techpool** [comment]
My changes 😆

**Comment by mbrandau** [comment]
@techpool Yeah, I do. :smiley: Just have a look at my comments.

**Pull request merged**